### DISCUSSION TITLE
Big thank you for all the work

### DISCUSSION THREAD

**Discussion by uros76** [seaky/nspanel_pro_tools_apk#163] [General]
I have 3 small pro panels and recently got one bigger pro panel. The latest one did not want to take in the xposed method of webview app instalation. It was almost a dead end for it. 
But then after a lot of research, I finally came to this place with all the latest info on how to use these great nspanel pro's other than stock sonoff stuff. 
So Robert, keep up the good work for all the community out there. Token of appreciation on your way via Paypal 😉

**Comment by seaky** [comment]
Thank you for the recognition! Sometimes people forget that these things are done not as a livelihood but in our free time, which is why such positive comments are especially appreciated!
